core/tests/protractor_utils/AdminPage.js

```diff
@@ -152,6 +159,50 @@ var AdminPage = function() {
       'Could not set role successfully');
     return true;
   };
+
+  this.viewRolesbyRole = function(role) {
+    waitFor.visibilityOf(roleDropdown,
+      'View role dropdown taking too long to be visible');
+    roleDropdown.sendKeys('By Role');
+
+    roleValueOption.click();
+    roleValueOption.sendKeys(role);
+
+    viewRoleButton.click();
+    return true;
```

Comment:
I think I agree with DubeySandeep here... Why does this return true?

Reply:
Isn't required..Removed that.